.docker/test.ini

```diff
@@ -0,0 +1,272 @@
+ckan.harvest.mq.type = redis
+ckan.harvest.mq.hostname = redis
+ckan.harvest.mq.port = 6379
+ckan.harvest.mq.redis_db = 0
+
+
+### ckanext-validation
+##Use the following configuration options to choose the operation modes:
+ckanext.validation.run_on_create_async = True
+ckanext.validation.run_on_update_async = True
+ckanext.validation.run_on_create_sync = False
+ckanext.validation.run_on_update_sync = False
+##Formats to validate
+##By default validation will be run against the following formats: CSV, XLSX and XLS. You can modify these formats using the following option:
+ckanext.validation.formats = csv xlsx xls
+##You can also provide validation options that will be used by default when running the validation:
+
+## ckanext-scheming settings
+#   module-path:file to schemas being used
+#scheming.dataset_schemas = ckanext.data_qld:ckan_dataset.json
+scheming.dataset_schemas = ckanext.data_qld:ckan_dataset.json ckanext.harvester_data_qld_geoscience:geoscience_dataset.json
+#   will try to load "spatialx_schema.json" and "spatialxy_schema.json"
+#   as dataset schemas
+#
+#   URLs may also be used, e.g:
+#
+# scheming.dataset_schemas = http://example.com/spatialx_schema.json
+
+#   Preset files may be included as well. The default preset setting is:
+scheming.presets = ckanext.scheming:presets.json ckanext.data_qld:presets.json
+
+#   The is_fallback setting may be changed as well. Defaults to false:
+scheming.dataset_fallback = false
+
+## ckanext-datarequests settings
+# Enable or disable the comments system by setting up the ckan.datarequests.comments property in the configuration file (by default, the comments system is enabled).
+ckan.datarequests.comments = true
+# Enable or disable a badge to show the number of data requests in the menu by setting up the ckan.datarequests.show_datarequests_badge property in the configuration file (by default, the badge is not shown).
+ckan.datarequests.show_datarequests_badge = true
+# Enable or disable description as a required field on data request forms
+ckan.datarequests.description_required = true
+# Default organisation used for new data requests
+ckan.datarequests.default_organisation = open-data-administration-data-requests
+# Enable or disable circumstances for closing data requests. Default value is False
+ckan.datarequests.enable_closing_circumstances = True
+
+# YTP Comments
+ckan.comments.moderation = False
+ckan.comments.moderation.first_only = False
+ckan.comments.threaded_comments = True
+ckan.comments.users_can_edit = False
+ckan.comments.check_for_profanity = True
+ckan.comments.bad_words_file = /app/ckan/default/src/ckanext-ytp-comments/ckanext/ytp/comments/bad_words.txt
+ckan.comments.show_comments_tab_page = True
+
+# Archiver
+ckanext-archiver.archive_dir = /app/filestore/archive
+ckanext-archiver.cache_url_root = http://dataqld-ckan.docker.amazee.io/resources/
+
+# QA
+qa.resource_format_openness_scores_json = /app/ckanext/data_qld/resource_format_openness_scores.json
```

Comment:
This is not the correct path for the QA config, when the current extension is not ckanext-data-qld.